src/main/java/org/gridsuite/loadflow/server/service/LoadFlowWorkerService.java

```diff
@@ -143,6 +144,60 @@ private void cleanLoadFlowResultsAndPublishCancel(UUID resultUuid, String receiv
         notificationService.publishStop(resultUuid, receiver);
     }
 
+    private static LoadingLimits.TemporaryLimit handleEquipmentLimitViolation(Branch<?> branch, LimitViolationInfos violationInfo) {
```

Comment:
- could we find a better describing name for this method ?   is it getBranchLowerLimitViolation() ? 
- can we add a comment somewhere to say:
              // limits are returned from the store by DESC duration / ASC value